ios/iosApp/Domain/AppInformationImpl.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import Shared
+
+class AppInformationImpl: AppInformation {
```

Comment:
This should also be done using expect-actual instead of platform specific implementations

Reply:
The problem here is that on android, we need the Context object and access to the buildConfig of the app. If we do this in the `androidMain` in the `shared` module, we don't have access to any of those.

Reply:
I got a solution for this that I'll put on main branch 🙂